src/libstd/fs/mod.rs

```diff
@@ -391,35 +484,85 @@ impl AsInnerMut<fs_imp::OpenOptions> for OpenOptions {
 
 impl Metadata {
     /// Returns whether this metadata is for a directory.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::fs::File;
+    ///
+    /// # fn foo() -> std::io::Result<()> {
+    /// let mut f = try!(File::create("foo.txt"));
+    /// let metadata = try!(f.metadata());
+    ///
+    /// assert_eq!(false, metadata.is_dir());
```

Comment:
Out of curiosity, why not `assert!(!metadata.is_dir())`? I thought the left was "actual" and right was "expected" as well most of the time in `assert_eq`, although I don't think we have a convention one way or another.